frontend/src/muralForm/MuralForm.tsx

```diff
@@ -87,22 +102,30 @@ function MuralForm({ handleCancel }: IMuralFormProps) {
       alert("Please check validity of address.");
       return;
     }
-    axios
-      .post(CREATE_MURAL_API, {
-        name: name,
-        boroughId: borough,
-        artistId: artist,
-        year: year,
-        city: "Montreal",
-        longitude: addressCoords[0],
-        latitude: addressCoords[1],
-        assistants: assistants,
-        partners: partners,
-        description: description,
-        socialMedia: socialMedia,
-        address: address,
-        neighbourhood: neighbourhood,
-      })
+
+    let payload = {
```

Comment:
this should be moved to an interfaces directory right?

Reply:
yep good call